tests/lib/test-utils.ts

```diff
@@ -1,14 +1,49 @@
 import { resolve } from 'path';
 import { TSESLint } from '@typescript-eslint/experimental-utils';
 
+const DEFAULT_TEST_CASE_CONFIG = {
+  filename: 'MyComponent.test.js',
+};
+
+class TestingLibraryRuleTester extends TSESLint.RuleTester {
```

Comment:
Our custom RuleTester just injects default test case config for each valid/invalid test case so we don't need to set `filename` config to a valid value to be reported.